### PR TITLE
initial commit for gradio app

### PR DESCRIPTION
**Pull request by deep-diver**
Here is the Space at HF Hub
: https://huggingface.co/spaces/chansung/segformer-tf-transformers

### CODE REVIEW

File: apps/gradio/app.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+import numpy as np
+from PIL import Image
+import tensorflow as tf
+from transformers import SegformerFeatureExtractor, TFSegformerForSemanticSegmentation
+
+feature_extractor = SegformerFeatureExtractor.from_pretrained(
+    "nvidia/segformer-b5-finetuned-ade-640-640"
+)
+model = TFSegformerForSemanticSegmentation.from_pretrained(
+    "nvidia/segformer-b5-finetuned-ade-640-640", from_pt=True
```
Comment (sayakpaul) [inline apps/gradio/app.py:12]: Let's assign this to a variable called `model_checkpoint`?

File: apps/gradio/app.py [hunk 2]
```diff
@@ -0,0 +1,201 @@
+import gradio as gr
+
+import numpy as np
+from PIL import Image
+import tensorflow as tf
+from transformers import SegformerFeatureExtractor, TFSegformerForSemanticSegmentation
+
+feature_extractor = SegformerFeatureExtractor.from_pretrained(
+    "nvidia/segformer-b5-finetuned-ade-640-640"
```
Comment (sayakpaul) [inline apps/gradio/app.py:9]: Same.

File: apps/gradio/requirements.txt [hunk 3]
```diff
@@ -2,5 +2,6 @@ numpy==1.23.0
 pillow==9.1.1
 matplotlib==3.5.2
 tensorflow==2.9.0
+torch==1.12.0
 git+https://github.com/sayakpaul/transformers.git@tf-segformer
```
Comment (sayakpaul) [inline apps/gradio/requirements.txt:6]: My god. It works?

**Review by sayakpaul — CHANGES_REQUESTED** [review]
LGTM with minor comments.

Also wondering if it makes sense to also show the labels of the segmentation maps as done [here](https://github.com/sayakpaul/Adventures-in-TensorFlow-Lite/blob/master/DeepLabV3/DeepLab_TFLite_ADE20k.ipynb).

**Comment by deep-diver** [comment]
@sayakpaul 

changed the variable names
: `feature_extractor` ==> `feature_checkpoint`
: `model` ==> `model_checkpoint`

**Comment by sayakpaul** [comment]
Makes sense to just keep it model_checkpoint since they are both essentially same.

**Comment by deep-diver** [comment]
@sayakpaul 

so the last change resolves the naming issue?

**Comment by sayakpaul** [comment]
Yes. 

I can take care of https://github.com/deep-diver/segformer-tf-transformers/pull/3#pullrequestreview-1026799708

**Comment by deep-diver** [comment]
Oh I missed that one. Please do that instead of me othewise it will take some time (I am not available for now)

**Pull request merged**